common/scripts/upgrade_docker.sh

```diff
@@ -25,161 +42,124 @@ __remove_services() {
   fi
 }
 
-__create_install_docker_script() {
-  if $DOCKER_UPGRADE_REQUIRED; then
-    __process_msg "Creating docker install script"
-
-    rm -f $INSTALL_DOCKER_SCRIPT
-    touch $INSTALL_DOCKER_SCRIPT
-    echo '#!/bin/bash' >> $INSTALL_DOCKER_SCRIPT
-    echo 'install_docker_only="true"' >> $INSTALL_DOCKER_SCRIPT
-    echo "SHIPPABLE_HTTP_PROXY=\"$SHIPPABLE_HTTP_PROXY\"" >> installDockerScript.sh
-    echo "SHIPPABLE_HTTPS_PROXY=\"$SHIPPABLE_HTTPS_PROXY\"" >> installDockerScript.sh
-    echo "SHIPPABLE_NO_PROXY=\"$SHIPPABLE_NO_PROXY\"" >> installDockerScript.sh
-
-    local node_scripts_location=/tmp/node
-    local node_s3_location="https://s3.amazonaws.com/shippable-artifacts/node/$RELEASE/node-$RELEASE.tar.gz"
-
-    pushd /tmp
-    mkdir -p $node_scripts_location
-    wget $node_s3_location
-    tar -xzf node-$RELEASE.tar.gz -C $node_scripts_location --strip-components=1
-    rm -rf node-$RELEASE.tar.gz
-    popd
-
-    cat $node_scripts_location/lib/logger.sh >> $INSTALL_DOCKER_SCRIPT
-    cat $node_scripts_location/lib/headers.sh >> $INSTALL_DOCKER_SCRIPT
-    cat $node_scripts_location/initScripts/$ARCHITECTURE/$OPERATING_SYSTEM/Docker_"$DOCKER_VERSION".sh >> $INSTALL_DOCKER_SCRIPT
-
-    rm -rf $node_scripts_location
-    # Install Docker
-    chmod +x $INSTALL_DOCKER_SCRIPT
-  fi
-}
+__update_docker_on_swarm_workers() {
+  __process_msg "Upgrading docker version on swarm workers"
```

Comment:
this will be new functionality. i dont think swarm workers' proxy settings were updated on every upgrade before. is this ok?

Reply:
we were supposed to do that. it doesnt make sense to just update master and leave the workers with old settings. i'd say this change is fixing a bug rather than introducing a new funtionality